notebook.ipynb

```diff
@@ -992,19 +1167,19 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 83,
+   "execution_count": 177,
    "metadata": {},
    "outputs": [],
    "source": [
-    "# # Convert AcademincLevel values to Undergraduate and Postgraduate.\n",
+    "# Convert AcademicLevel values to Undergraduate and Postgraduate.\n",
     "\n",
-    "# data['AcademicLevel'] = data['AcademicLevel'].replace(['Graduation', 'PhD', 'Master', '2n Cycle'], \"Post Graduate\")\n",
-    "# data['AcademicLevel'] = data['AcademicLevel'].replace([\"Basic\"], \"Under Graduate\")\n"
+    "data['AcademicLevel'] = data['AcademicLevel'].replace(['Graduation', 'PhD', 'Master', '2n Cycle'], \"Post Graduate\")\n",
```

Comment:
Also, this conversion should be left in comments, else it will change the feature `AcademicLevel` which will make it unsuitable for the Bivariate Analysis which you did later.